### ISSUE TITLE
NPCs clear themselves

### ISSUE THREAD

**Issue by MrBlasyMSK** [brokiem/SimpleNPC#66]
**Describe the bug**
NPCs are erased automatically, but it only happens in a part of the map.

**To Reproduce**

1. I don't know how to reproduce it since it is random and only occurs in one area of the map

**Expected behavior**
That they will not be erased, because it does not make any sense, they have never done it before

**Screenshots**
![image](https://user-images.githubusercontent.com/85447029/126917808-130bae9b-fab5-4dbd-8f03-bfa663ba6d69.png)
It is those 3 npcs that disappear

**SimpleNPC plugin information**

- Version: Dev288
- Server api: 3.22.1

**Additional context**
This has never happened to me, neither with simplenpc before nor with Slappers, it is very rare.
All other NPCs work properly and are not eliminated

**Comment by MrBlasyMSK** [comment]
I just found the way to erase

They are only cleaned in that part, if I restart 2 times in a row (and nobody enters the server) they are automatically deleted, without reason

I don't have any clear explanation to that but hopefully you can fix it

**Comment by brokiem** [comment]
No, try /save-all before stopping the server

**Comment by MrBlasyMSK** [comment]
I just did it now, I'll let you know if it happens to me again

**Comment by MrBlasyMSK** [comment]
It keeps happening, if you do a restart twice and nobody enters the lobby, those 3 dolls are automatically deleted

@brokiem

**Comment by quydrop** [comment]
That's right, i am having the same problem in my Server, please fix it!!! 
@brokiem

**Comment by MrBlasyMSK** [comment]
The problem is if you have the BlockHunt plugin or it was the TNTRun I don't remember, but if you have those plugins it will happen to you, since I eliminated them it no longer happens to me

> That's right, i am having the same problem in my Server, please fix it!!!
> @brokiem

**Comment by quydrop** [comment]
> The problem is if you have the BlockHunt plugin or it was the TNTRun I don't remember, but if you have those plugins it will happen to you, since I eliminated them it no longer happens to me
> 
> > That's right, i am having the same problem in my Server, please fix it!!!
> > @brokiem

My server doesn't use those Plugins, I only have Entity related Plugins like Boss, Decorations. In addition, when there are 2 NPCs in the same World, it only loses 1 NPC if restarting SV consecutively without losing all NPCs

**Comment by MayCantCode** [comment]
same happened to me so try deleting each plugin one by one and try . if they dont get deleted after a plugin is removed then that plugin is causing the problem . a plugin was causing it for me
